localstack-core/localstack/config.py

```diff
@@ -1321,6 +1321,7 @@ def use_custom_dns():
     "LS_LOG",
     "MAIN_CONTAINER_NAME",
     "MAIN_DOCKER_NETWORK",
+    "MSSQL_ACCEPT_EULA",
```

Comment:
would it be possible to add this config variable in Pro instead? or does it not work with new emulators?

Reply:
Ah, that should work as well! I didn't know that it was a Pro feature in AWS. I've opened a new PR there instead. :+1: